WalletWasabi.Fluent/Views/CoinSelection/Core/Cells/CoinSelectionCellView.axaml.cs

```diff
@@ -0,0 +1,16 @@
+using Avalonia.Controls;
+using Avalonia.Markup.Xaml;
+
+namespace WalletWasabi.Fluent.Views.CoinSelection.Core.Cells;
+public class CoinSelectionCellView : UserControl
```

Comment:
trivial: mind the blank line between namespace declaration and class declaration, for the sake of consistency.